Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -541,6 +541,7 @@ ioi ld   hl,(hl)           ; 13
 	; Less 6 to account for the status vector itself
    ld    hl,MAX_MTU+MAX_OVERHEAD-6
    sub   hl,de    ; HL = length of last buffer
+   push  jkhl	  ; save that length part for later

```

Comment:
`jk` is an unknown value here -- can we do a `push hl` and then later on 558 `pop de` so it's clear what you're doing?